datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -60,17 +60,16 @@ STORED AS parquet
 LOCATION 'test_files/scratch/insert_to_external/parquet_types_partitioned/'
 PARTITIONED BY (b);
 
-query I
+query error
 insert into dictionary_encoded_parquet_partitioned
 select * from dictionary_encoded_values
 ----
-2
+DataFusion error: Error during planning: Inserting query must have the same schema nullability as the table. Expected table field 'b' nullability: false, got field: 'b', nullability: true
```

Comment:
This is also expected, because PARTITIONED BY (b), will make the b nullable to false.

We shouldn't support insert nullable value for partition key i think.

Reply:
`because PARTITIONED BY (b), will make the b nullable to false` can be add as comment

Reply:
Added comments in latest PR, thanks!

Error should be after query error to pass CI, i think it's auto generated after the PR here:

https://github.com/apache/datafusion/pull/14439

Reply:
It is auto generated a long time before, we need to manually move it

Reply:
@jayzhan211  This https://github.com/apache/datafusion/pull/14439/files#diff-51757b2b1d0a07b88551d88eabeba7f74e11b5217e44203ac7c6f613c0221196R273 merged less than a week ago. I think it's start from there, and i tried local sql logic test use -- --complete, it's also the same result, i guess the CI also use it to generate and verify?

Reply:
This is the multiline error feature of sqllogictest-rs.
https://github.com/risinglightdb/sqllogictest-rs/blob/7ee44cd995fb65175bb647d07f63b557dbaa22c7/CHANGELOG.md#0180---2023-11-08